aws/sdk-adhoc-test/build.gradle.kts

```diff
@@ -29,6 +29,11 @@ buildscript {
     dependencies {
         classpath("software.amazon.smithy:smithy-cli:$smithyVersion")
     }
+    repositories {
```

Comment:
I think you can just put this in `settings.gradle.kts` once: https://docs.gradle.org/current/userguide/declaring_repositories.html#sub:centralized-repository-declaration